### ISSUE TITLE
Can't use ShinraMeter with WTFast

### ISSUE THREAD

**Issue by Tailsview** [neowutran/ShinraMeter#123]
I'm unable to use ShinraMeter with WTFast, it keeps giving me the No server prompt. Please someone help :(

**Comment by Gl0** [comment]
https://github.com/neowutran/ShinraMeter/wiki#vpn

**Comment by Tailsview** [comment]
You'll have to dumb it down for me.

**Comment by Gl0** [comment]
What are you misunderstanding?

**Comment by Tailsview** [comment]
Everything I guess, where do I get the IP from, what format do I type it in. Just yeah everything.

**Comment by Gl0** [comment]
While tera is connected to game server via wtfast:
Open Resource Monitor ( `perfmon /res` from command prompt or via button in task manager).
Open network tab, expand TCP connections list.
Search for Tera.exe and check "Remote address" column. This is ip that you need.
Open servers.txt in resources\data subdir with notepad
Add line at the end:
```123.45.67.89 KR  VPN```
change 123.45.67.89 to your actual wtfast ip founded earlier.
change region to your region. Notice 2 spaces between region and "VPN", github eating it in comments.
Save and close file.
Check that config\windows.xml contains `<winpcap>false</winpcap>`, if there is `<winpcap>true</winpcap>` - change `true` to `false`, save and close.
Run add_firewall_exception.bat with admin privileges or [add firewall rule manually](https://github.com/neowutran/ShinraMeter/wiki/Window-firewall).
Start meter. Restart tera. Profit.

**Comment by Tailsview** [comment]
What if the Remore Address says IPv4 loopback?

**Comment by Gl0** [comment]
use 127.0.0.1

**Comment by Tailsview** [comment]
So it should look like this 127.0.0.1 AU VPN

**Comment by Gl0** [comment]
but with wtfast AFAIK there should be public IP of wtfast servers. May be they changed something.
We don't have AU region.

**Comment by Tailsview** [comment]
-Sigh- I can't get this thing to work at all.. Since adding the IP into the server list Shinra wont' even load :/

**Comment by Gl0** [comment]
127.0.0.1 EU  VPN
with 2 spaces between EU and VPN

**Comment by Tailsview** [comment]
All right it's now saying that the server is VPN though when I do DPS ingame the calculator doesn't kick in.

**Comment by Gl0** [comment]
It should change to your normal server name after login

**Comment by Tailsview** [comment]
Nope still says VPN

**Comment by Gl0** [comment]
Are you really connecting to EU server, may be some other region?

**Comment by Tailsview** [comment]
US Maybe?

**Comment by Gl0** [comment]
You should know, where you are playing =) If it's US - there should be US.=)

**Comment by Tailsview** [comment]
208.67.49.84 NA 4024 Ascension Valley <-- That's my server. So just change EU to NA?

**Comment by Gl0** [comment]
Yes, of cause )

**Comment by Tailsview** [comment]
It's beautiful!

**Comment by Tailsview** [comment]
Now for the last part. How can I use the calculator in Fullscreen?

**Comment by Gl0** [comment]
You can use it, but  you'll not see it until you alt-tab from game or paste stats into game chat.
